packages/visual-editor/src/hooks/useComponent.tsx

```diff
@@ -7,10 +7,13 @@ import type {
 import { useMemo } from 'react';
 import { useComponentProps } from './useComponentProps';
 import { builtInComponents } from '@/types/constants';
-import { DESIGN_COMPONENT_NODE_TYPE } from '@contentful/experience-builder-core/constants';
-import { ContentfulContainer, DesignComponent } from '@contentful/experience-builder-components';
-import { resolveDesignComponent } from '@/utils/designComponentUtils';
-import { componentRegistry, createDesignComponentRegistration } from '@/store/registries';
+import {
+  DESIGN_COMPONENT_NODE_TYPE,
+  ASSEMBLY_NODE_TYPE,
```

Comment:
Has `ASSEMBLY_NODE_TYPE` been added in `@contentful/experience-builder-core`? Will this cause any errors if not on the latest version of the SDK?

Reply:
Yes, you can see this in die diff at `/core/constants.ts` (all constants are exported from this file)